### PR TITLE
user-guide: added nodes requirements

### PR DESCRIPTION
**Pull request by danielerez**

### CODE REVIEW

File: docs/user-guide.md [hunk 1]
```diff
@@ -170,7 +170,7 @@ INFO Download openshift-install from: https://mirror.openshift.com/pub/openshift
 ### Generate a Cluster Configuration Image
 * Create a configuration directory
 
-**:warning: use absolute paths.**
+**:warning: Use absolute directory paths.**
```
Comment (nmagnezi) [inline docs/user-guide.md:173]: Please change line 52 as well.

Comment (danielerez) [reply docs/user-guide.md:173]: done

**Review by nmagnezi — APPROVED** [review]
/lgtm

**Pull request merged**